### PR TITLE
0.4.0 Release

### PR DESCRIPTION
**Pull request by BurnzZ**
TODO:

- [x] update date in `CHANGES.rst`
- [ ] Merge https://github.com/zytedata/zyte-spider-templates-project/pull/7 after this PR is merged
- [ ] Tag `0.4.0`

### CODE REVIEW

**Review by wRAR — APPROVED** [review]
Will bump2version work correctly when the versions in the source are already bumped?

**Comment by PyExplorer** [comment]
Looks good for me! @BurnzZ is this feature https://github.com/zytedata/zyte-spider-templates-project/pull/7 to use in Articles?

**Comment by BurnzZ** [comment]
> is this feature https://github.com/zytedata/zyte-spider-templates-project/pull/7 to use in Articles?

@PyExplorer it was mainly for eCommerce websites that aggregates products across multiple websites. This could also be used in Articles, yes.

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
 
+0.4.0 (YYYY-MM-DD)
+------------------
+
+* Products outside of the domain can now be crawled using
+  ``zyte_spider_templates.middlewares.AllowOffsiteMiddleware``.
+
+* Updated documentation to also setup ``zyte_common_items.ZyteItemAdapter``.
+
+* Improved ``max_requests`` spider parameter description.
```
Comment (Gallaecio) [inline CHANGES.rst:12]:
```suggestion
* Products outside of the target domain can now be crawled using
  :class:`zyte_spider_templates.middlewares.AllowOffsiteMiddleware`.

* Updated the documentation to also set up ``zyte_common_items.ZyteItemAdapter``.

* Improved the description of the ``max_requests`` spider parameter.
```

I am not 100% sure of whether or not `:class:` will make the doc build fail if the class is not in the docs, so run `tox -e docs` to make sure, and revert if it causes the build to fail. Or add the class somewhere in the docs :slightly_smiling_face:.

**Review by Gallaecio — APPROVED** [review]

File: CHANGES.rst (hunk 1 above)
Comment (BurnzZ) [reply CHANGES.rst:12]: It does work! 🙌

**Pull request merged**